README.md

```diff
@@ -51,6 +51,23 @@ Or toasters on amazon.
 s -p am toasters
 ```
 
+## Provider Autocompletion
+
+Autocompletion for providers is supported. For setting up autocompletion, 
+
+1. Have `s` installed
+2. Add the following lines to `~/.bash_profile`
+```
+if [ -f /path/to/s-completion.bash ]; then
+    . /path/to/s-completion.bash
```

Comment:
In your PR add this to the completion file

```
if [[ -n ${ZSH_VERSION-} ]]; then
  autoload -U +X bashcompinit && bashcompinit
fi
```

This adds full zsh support, tested and working. We should update the docs and comments to reflect this.